src/main/java/frc/robot/Claw.java

```diff
@@ -0,0 +1,68 @@
+package frc.robot;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+import com.ctre.phoenix.motorcontrol.ControlMode;
+
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+import edu.wpi.first.wpilibj.DoubleSolenoid.Value;
+
+/**
+ * Claw
+ * 
+ * The code for the claw will be posted here for use on the new robot.
+ * 
+ * @author (Charlie Bergdall)
+ * @version (0.0.1) Swenson's Claw uses Neos (Spark MAX), solenoids,
+ */
+public class Claw {
+    //Global Variables
+    public static DoubleSolenoid LPiston, RPiston;
+    public static CANSparkMax claw;
+    public static TalonSRX clawIntakeMotor;
+    private static double scaler = 0.25;
```

Comment:
Does this ever plan on changing during runtime? (is it going to stay constant)? If so what keyword modifier should we use to indicate this variable as such.

Reply:
set the value as FINAL. I'll add that now.